tests/integration/test_disks_app_func/test.py

```diff
@@ -119,6 +119,24 @@ def test_disks_app_func_cp(started_cluster):
 
     assert "path1" in out
 
+    source.exec_in_container(
+        [
+            "/usr/bin/clickhouse",
+            "disks",
+            "--send-logs",
+            "--disk",
+            "test2",
+            "remove",
+            "path1",
+        ]
+    )
+
+    out = source.exec_in_container(
```

Comment:
Why we need list one more time?

Reply:
it is an assertion that the file is indeed deleted